style.css

```diff
@@ -0,0 +1,30 @@
+* {
+	margin: 0;
+	padding: 0;
+}
+
+html {
+	height: 100%;
+}
+
+body {
+	background-color: coral;
+}
```

Comment:
Coral is a very bold choice for the web page. Have you tested with other colors especially considering color blind users?